src/lib.rs

```diff
@@ -17,14 +18,21 @@ mod circuit;
 mod constants;
 mod nifs;
 mod r1cs;
-
 // public modules
 pub mod errors;
 pub mod gadgets;
 pub mod provider;
 pub mod spartan;
 pub mod traits;
 
+// experimental modules
+#[cfg(feature = "hypernova")]
+mod ccs;
+#[cfg(feature = "hypernova")]
+mod hypercube;
+#[cfg(feature = "hypernova")]
+mod utils;
```

Comment:
Could we move this to the ccs module as well as this is used only by the hypernova code?